### PR TITLE
FamilyMemberHistory Updates

### PR DESCRIPTION
**Pull request by sydneanschutz**
Description
----
FamilyMemberHistory Updates

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screen Shot 2021-09-07 at 2 45 37 PM
Screen Shot 2021-09-07 at 2 45 42 PM
Screen Shot 2021-09-07 at 2 45 47 PM
Screen Shot 2021-09-07 at 2 46 01 PM
Screen Shot 2021-09-07 at 2 46 09 PM
Screen Shot 2021-09-07 at 2 46 14 PM

### CODE REVIEW

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 1]
```diff
@@ -15,10 +15,74 @@ The FamilyMemberHistory resource represents family history for a given patient.
 
 The resource should be leveraged as a snapshot in time and new data should be consistently retrieved through the API rather than stored within an application. There may be instances where a relative had an associated condition removed from the patient’s chart which would then no longer be exposed on a subsequent transaction. It is crucial that the data be refreshed with every interaction.
 
+It is recommended that an update to FamilyMemberHistory contain all relevant data that previously existed on the resource to ensure that data remains populated in the patient's chart (e.g., FamilyMemberHistory.sex, FamilyMemberHistory.bornDate, FamilyMemberHistory.condition.onsetDate, etc.). The absence of this optional information may result in the removal of data from the patient's chart.
+
```
Comment (sydneanschutz) [inline content/millennium/r4/clinical/summary/family-member-history.md:19]: the jira also says:
Implementation Details - Need to add something along the lines of:

During a client implementation, confirm what vocabulary is supported. Best practice is to leverage a single vocabulary and the Model recommendation is for clients to leverage SNOMED-CT (https://wiki.cerner.com/display/public/reference/Configure+Family+History)

Not sure if we need to say that here?

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 2]
```diff
@@ -15,10 +15,74 @@ The FamilyMemberHistory resource represents family history for a given patient.
...
 The following fields are returned if valued:
 
-* [FamilyMemberHistory id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
- 
+* [Id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.status){:target="_blank"}
```
Comment (MadhurRaj) [inline content/millennium/r4/clinical/summary/family-member-history.md:23]: all these need to be https.

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 3]
```diff
@@ -35,8 +99,9 @@ Search for FamilyMemberHistories that meet supplied query parameters:
 
 Name      | Required?       | Type          | Description
 ----------|-----------------|---------------|-------------------------------------------------------
-`_id`     | This or patient | [`token`]     | The logical resource id associated with the resource.
-`patient` | This or _id     | [`reference`] | The identity of a subject to list family member history items for. Example: `12345`
+`_id`     | This or patient         | [`token`]     | The logical resource id associated with the resource.
+`patient` | This or _id             | [`reference`] | The identity of a subject to list family member history items for. Example: `12345`
+`status`  | No. Patient is required | [`token`] | The status of the record of the family history of a specific family member.
```
Comment (MadhurRaj) [inline content/millennium/r4/clinical/summary/family-member-history.md:104]: I think we just leave this as No

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 4]
```diff
@@ -35,8 +99,9 @@ Search for FamilyMemberHistories that meet supplied query parameters:
...
```
Comment (MadhurRaj) [inline content/millennium/r4/clinical/summary/family-member-history.md:105]: let's add a notes section here and say that:

`status` may be provided when the `patient` search parameter is provided.

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 5]
```diff
@@ -99,6 +164,7 @@ Create a new FamilyMemberHistory.
 _Implementation Notes_
 
 * Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* Conditions can only be added through the PUT operation.
```
Comment (MadhurRaj) [inline content/millennium/r4/clinical/summary/family-member-history.md:167]: let's keep this specific to creates...

Conditions cannot be added while creating the FamilyMemberHistory.

File: lib/resources/r4/family_member_history.yaml [hunk 6]
```diff
@@ -1,28 +1,553 @@
...
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+            "code": "BRO"
+          }
+        ]
+      }
+    }
+  note: |
+    <ul>
+      <li>Should we explain what we are using 'FAMMEMB' code for?</li>
```
Comment (MadhurRaj) [inline lib/resources/r4/family_member_history.yaml:146]: yea... maybe something like:

a relationship of FAMMEMB indicates a FamilyMemberHistory that contains information specific to the patient that pertains to all relations of the patient.

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 2 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md]: https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68L23

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 3 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md]: https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68R108

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 4 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md:113]: https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68R110

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 5 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md]: https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68L167

File: lib/resources/r4/family_member_history.yaml (hunk 6 above)
Comment (sydneanschutz) [reply lib/resources/r4/family_member_history.yaml]: https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63L146

File: lib/resources/r4/family_member_history.yaml [hunk 7]
```diff
@@ -1,28 +1,554 @@
...
+  required: 'Yes'
+  description: Indication of the precision of a given value.
+  url: https://hl7.org/fhir/extensibility.html#Extension
+  action:
+    - create
+    - update
+    - terminology
+  note: |
+    <ul>
+      <li> Supported codes include 397669002, 26175008, 288563008, 272113006 </li>
```
Comment (RayGunY) [inline lib/resources/r4/family_member_history.yaml:257]: Do we want to specify what these codes stand for?

Comment (MadhurRaj) [reply lib/resources/r4/family_member_history.yaml:257]: Hmm...I don't think we should specify which codes are supported. Let's just list that only snomed codes are supported. We can be more particular if needed in the future.

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 4 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md:113]: https://github.com/cerner/fhir.cerner.com/pull/728/commits/2e9e8da0d0db2c5390c8c1fed4efe0d224b4b043#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68L112

File: lib/resources/r4/family_member_history.yaml (hunk 7 above)
Comment (sydneanschutz) [reply lib/resources/r4/family_member_history.yaml]: removed https://github.com/cerner/fhir.cerner.com/pull/728/commits/2e9e8da0d0db2c5390c8c1fed4efe0d224b4b043#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63L255

**Review by MadhurRaj — APPROVED** [review]

**Review by RayGunY — APPROVED** [review]

**Comment by ashu-bdvt** [comment]
Post deployment screenshots captured:

Screenshot 2021-09-09 at 7 53 36 PM
Screenshot 2021-09-09 at 7 53 24 PM
Screenshot 2021-09-09 at 7 53 06 PM
Screenshot 2021-09-09 at 7 52 48 PM
Screenshot 2021-09-09 at 7 52 10 PM
Screenshot 2021-09-09 at 7 51 54 PM

**Pull request merged**